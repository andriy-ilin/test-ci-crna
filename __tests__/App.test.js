import "react-native";
import React from "react";
import { Text } from "react-native";
import { shallow } from "enzyme";
import NavigationTestUtils from "react-navigation/NavigationTestUtils";

describe("App snapshot", () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it("Text style", async () => {
    const wrapper = shallow(
      <Text
        style={{ color: "red" }}
        justifyContent="flex-end"
        alignContent="center"
        alignItems="center"
      >
        test
      </Text>
    );
    expect(wrapper.prop("style")).toMatchSnapshot();
  });
});
