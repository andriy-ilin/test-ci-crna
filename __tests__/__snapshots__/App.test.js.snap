// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot Text style 1`] = `
Object {
  "color": "red",
}
`;
